src/main/java/eu/xenit/gradle/alfrescosdk/AmpPlugin.java

```diff
@@ -22,10 +22,10 @@ public void apply(Project project) {
         project.getConfigurations().create(AMP_CONFIGURATION);
         project.getArtifacts().add(AMP_CONFIGURATION, amp);
 
-        amp.setModuleProperties(ampConfig.getModulePropertiesSupplier());
-        amp.setConfig(ampConfig.getConfigDirSupplier());
-        amp.setWeb(ampConfig.getWebDirSupplier());
-        amp.setFileMappingProperties(ampConfig.getFileMappingPropertiesSupplier());
+        amp.setModuleProperties(ampConfig::getModuleProperties);
```

Comment:
@Jasper-Hilven This is much nicer for sure, but was it broken?

Reply:
Yes, the unit test failed before the fix.

Reply:
Initial suppliers were passed to the task, and when setting a new supplier via the ampConfig this was not passed through again.